artiq/gateware/drtio/transceiver/gtx_7series.py

```diff
@@ -334,6 +336,11 @@ def __init__(self, clock_pads, pads, sys_clk_freq, rtio_clk_freq=125e6, master=0
             self.cd_rtio.clk.eq(self.gtxs[master].cd_rtio_tx.clk),
             self.cd_rtio.rst.eq(reduce(or_, [gtx.cd_rtio_tx.rst for gtx in self.gtxs]))
         ]
+
+        self.comb +=[
```

Comment:
``self.comb += stable_clkin.eq(self.stable_clkin.storage | self.gtxs[0].tx_init.cplllock)`` is enough